backend/curriculum_tracking/serializers.py

```diff
@@ -354,8 +365,18 @@ class Meta:
             "open_pr_count",
             "oldest_open_pr_updated_time",
             "repo_url",
+            "total_number_of_negative_reviews",
```

Comment:
maybe a better name would be:

number_of_times_card_moved_to_review_feedback.

If this was really the total number of negative reviews then the call would be very different.